scripts/upload_sample.py

```diff
@@ -44,36 +48,40 @@ class ExecutionContext(BaseModel):
     token: TokenObject | None = None
 
 
+def _rel_to_abs_path(path: Path | str, base_path: Path) -> Path:
+    """Resolve relative paths to absolute.
+
+    if a path is relative, convert to absolute from the configs parent directory
```

Comment:
This comment makes sense to have in my eyes! This helps understanding and goes beyond repeating the function name.